hack/ci/push-binaries/push-binaries.sh

```diff
@@ -21,24 +21,29 @@ set -o xtrace
 # this script is invoked in GCB as the entrypoint
 # avoid prow potentially not being in the right working directory
 REPO_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/../../.." &> /dev/null && pwd -P)"
+# export REPO_ROOT for gcs_upload_version to use in subshells
+export REPO_ROOT
 cd "${REPO_ROOT}" &> /dev/null
 
 # pass through git details from prow / image builder
 if [ -n "${PULL_BASE_SHA:-}" ]; then
   export COMMIT="${PULL_BASE_SHA:?}"
 else
-  COMMIT="$(git rev-parse --short HEAD 2>/dev/null)"
+  COMMIT="$(git rev-parse HEAD 2>/dev/null)"
   export COMMIT
 fi
 
+# short commit is currently 8 characters
```

Comment:
fun fact this can change as the repo gets more commits
you may want to consider a fixed size. IIRC kubernetes ran into this